.github/workflows/add-mmi-reviewer-and-notify.yml

```
@@ -0,0 +1,54 @@
+name: Add MMI reviewer and notify via Slack
+
+on:
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  process-label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add reviewer when team-mmi label is added
+        if: contains(github.event.pull_request.labels.*.name, 'team-mmi')
+        uses: actions/github-script@v5
```

latest version is v7
```suggestion
        uses: actions/github-script@v7
```